backend/python-langchain/main.py

```diff
@@ -236,6 +246,24 @@ def send_message():
         input_key="question"
     )
 
+    # ************ MEDICAL TRIALS VECTOR ************
+    instructor_embeddings = HuggingFaceInstructEmbeddings(model_name = 'hkunlp/instructor-base', model_kwargs = {"device": "cuda"})
```

Comment:
@felipegm0911 do we need GPU?

Reply:
yes, the vector was created with GPU. I could check if we can use the same vector without GPU, but if it works, the response would be slower